core/domain/question_services.py

```diff
@@ -142,6 +142,26 @@ def get_questions_and_skill_descriptions_by_skill_ids(
     return questions, skill_descriptions, next_cursor
 
 
+def get_questions_by_skill_ids(question_count, skill_ids):
+    """Returns constant number of questions linked to the given skill ids.
+
+    Args:
+        question_count: int. The number of questions to return.
+        skill_ids: list(str). The ID of the skills to which the questions are
+            linked.
+
+    Returns:
+        list(Question). The list of constant number of questions linked to each
```

Comment:
A list containing a total of question_count questions. There are a roughly-equal number of questions for each of the given skill IDs.

(Btw, are there any other constraints/issues you should be calling out here? E.g. are these returned in any particular order? What happens if there are insufficient questions in the datastore for a given skill?)

Reply:
Done.